src/app/components/title/title.component.html

```diff
@@ -10,24 +10,4 @@ <h1>Charming Look</h1>
   <p>{{ title.mainData.address }}</p>
```

Comment:
now your title.component doesn't show data and causes errors in console and that's because of wrong data format in json file. you wrote data inside arrays, but your interface doesn't like that ;) you have to change your json or interface
<img width="1110" alt="image" src="https://github.com/iwonaive/appointy/assets/66222290/dbb3e34c-357d-44bb-a2ce-c41fee4d0ee3">